code/src/components/Secondhand.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+export const Secondhand = ({ secondHand, setSecondHand, step, setStep }) => {
+  const handleSecondHandChange = (event) => {
+    setSecondHand(event.target.value)
+  }
+
+  const goToNextQuestion = () => {
+    setStep(step + 1);
+  };
+
+  const handleKeyDown = (event) => {
+    if (event.keyCode === 13) {
+      goToNextQuestion();
+    }
+  }
+
+  return (
```

Comment:
I really like that you suggest an answer when the user dont write anything it makes the survey conversational and definitely something that i will keep in mind for other projects 💯